tools/importers/helpers/structures.ts

```diff
@@ -0,0 +1,41 @@
+export interface Dict<V> {
+	[key: string]: V;
+}
+
+export interface ITokenJsonObject {
+	id: string;
+	label: string;
+	value: string;
+}
+
+export interface IResultJsonObject {
+	id: string;
+	label: string;
+	value: string;
+}
+
+export interface ConfigFileObject {
+	destinationVariables: string;
+	destinationVariablesCss: string;
+	destinationVariablesCssJson: string;
+	destinationJson: string;
+	destination: string;
```

Comment:
are they all required or optional? I see config file doesn't define all fields, so something is wrong

Reply:
Fixed.